packages/app-store/zoomvideo/lib/VideoApiAdapter.ts

```diff
@@ -257,13 +269,23 @@ const ZoomVideoApiAdapter = (credential: CredentialPayload): VideoApiAdapter =>
       }
     },
     createMeeting: async (event: CalendarEvent): Promise<VideoCallData> => {
+      let userSettings: ZoomUserSettings | undefined;
+      try {
+        const filterResp = "default_password_for_scheduled_meetings,auto_recording";
+        const responseBody = await fetchZoomApi(`users/me/settings?custom_query_fields=${filterResp}`);
+        userSettings = zoomUserSettingsSchema.parse(responseBody);
+      } catch (err) {
+        console.error(err);
+        /* If user settings could not be retrieved due to error in like - api scope configurations,
+         just continue create meeting with default settings*/
+      }
```

Comment:
Since we're repeating this functionality, can we abstract this into its own function? Maybe we can call that function inside of translate event.

Also we have to add a new scope to our Zoom app. If a user has not reauthenticated then that means this will always throw an error. can we add proper logging if that is the case?

Reply:
sure will do